yarn.lock

```diff
@@ -1288,14 +1288,22 @@
     "@types/history" "*"
```

Comment:
add this file to the .gitattributes

yarn.lock	-diff -merge

so that it doesn't pop in code reviews expanded

Reply:
it already has it.